examples/crd-client/go.mod

```diff
@@ -0,0 +1,12 @@
+module agones.dev/agones/examples/crd-client
+
+go 1.13
+
+require (
+	agones.dev/agones v1.1.0
```

Comment:
Any thoughts on how we keep this up to date as we keep releasing Agones versions?

Reply:
Should this now be 1.2.0 ?